### ISSUE TITLE
IOS Hermes main.jsbundle 0 bytes after upload

### ISSUE THREAD

**Issue by mrobinsonFS** [getsentry/sentry-react-native#3234]
**OS:**
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

**Platform:**
- [x]  iOS
- [ ]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 5.7.1

**`react-native` version:** 0.72.1

**Are you using Expo?**
- [ ]  Yes
- [x]  No

**Are you using sentry.io or on-premise?**
- [x]  sentry.io (SaaS)
- [ ]  on-premise

**If you are using sentry.io, please post a link to your issue so we can take a look:**

[[Link to issue]
](https://fullscript.sentry.io/issues/3990823831/?environment=production&project=6475581&query=is%3Aunresolved&referrer=issue-stream&stream_index=0)

**Configuration:**
```
(`@sentry/react-native`)
init({
    debug: false,
    dsn: Config.SENTRY_DSN,
    /*
     * Adding `!__DEV__` here because you could create local __DEV__ builds of
     * staging and production. This will only enable sentry for release configurations
     * of staging and production.
     */
    enabled: ["staging", "production", "development"].includes(Config.ENV),
    enableNative: true,
    environment: Config.ENV,
    integrations,
    release: version,
    dist,
    tracesSampleRate: 1.0,
    appHangTimeoutInterval: 4,
  });
```

---
**I have following issue:**

I am following this [guide](https://docs.sentry.io/platforms/react-native/manual-setup/hermes/#custom-source-maps) to create custom source maps for react native with Hermes. After successfully uploading the source maps to Sentry, the main.jsbundle is 0 bytes. I'm not sure what affect this has, but when the file is created on my local, it has data (usually similar size to the map). I am seeing errors in my issues with Invalid Absolute Path URL, but I'm not sure if it's related or not. Just want to check if the bundle is supposed to have data

**Steps to reproduce:**
- Generate the Source Maps and upload to Sentry 
```
npx react-native bundle --platform ios --dev false --entry-file index.js --reset-cache --bundle-output main.jsbundle --sourcemap-output main.jsbundle.map --minify false

ios/Pods/hermes-engine/destroot/bin/hermesc -O -emit-binary -output-source-map -out=main.jsbundle.hbc main.jsbundle

rm -f main.jsbundle
mv main.jsbundle.hbc main.jsbundle

mv main.jsbundle.map main.jsbundle.packager.map
node node_modules/react-native/scripts/compose-source-maps.js main.jsbundle.packager.map main.jsbundle.hbc.map -o main.jsbundle.map 

sentry-cli releases files "test" upload-sourcemaps \
    --dist 123ABC \
    --strip-prefix `pwd` \
    main.jsbundle main.jsbundle.map
```

- Step 2
- Observe issues and source maps settings

**Actual result:**

![image](https://github.com/getsentry/sentry-react-native/assets/109227096/8cd2da2e-1746-483a-8351-ae583ccdf6fd)

**Expected result:**
Bundle should not be 0 bytes

**Comment by krystofwoldrich** [comment]
Hi,
thank you for the link to the issue.

`Invalid Absolute Path URL` is caused by a bug that was fixed in [RN SDK 5.1.1](https://github.com/getsentry/sentry-react-native/releases/tag/5.1.1), please upgrade to that or newer.

Also, double-check if you are using `EXTRA_PACKAGER_ARGS` or [`SOURCEMAP_FILE`](https://reactnative.dev/docs/sourcemaps#enable-source-maps-on-ios)  in your Bundle React Native build phase. You might need only the packager bundle and source map.

**Comment by mrobinsonFS** [comment]
> Hi, thank you for the link to the issue.
> 
> `Invalid Absolute Path URL` is caused by a bug that was fixed in [RN SDK 5.1.1](https://github.com/getsentry/sentry-react-native/releases/tag/5.1.1), please upgrade to that or newer.
> 
> Also, double-check if you are using `EXTRA_PACKAGER_ARGS` or [`SOURCEMAP_FILE`](https://reactnative.dev/docs/sourcemaps#enable-source-maps-on-ios) in your Bundle React Native build phase. You might need only the packager bundle and source map.

Thanks Krystof, I have upgraded our RN SDK and the next build we release will be using 5.7.1. I'll double check the build phase and post my results back

**Comment by mrobinsonFS** [comment]
Looks like the invalid path error is gone with our upgrade, thanks!
